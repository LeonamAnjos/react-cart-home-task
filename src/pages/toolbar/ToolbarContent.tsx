import React from "react";
import Caption from "../../components/Caption";

const ToolbarContent = () => {
  return <Caption variant="h6">{"Cart Home Task"}</Caption>;
};

export default ToolbarContent;
