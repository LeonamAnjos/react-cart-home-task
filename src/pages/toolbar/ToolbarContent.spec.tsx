import { render } from "@testing-library/react";
import ToolbarContent from "./ToolbarContent";

describe("<ToolbarContent />", () => {
  it("SHOULD render", () => {
    const { container } = render(<ToolbarContent />);

    expect(container).toMatchSnapshot();
  });
});
