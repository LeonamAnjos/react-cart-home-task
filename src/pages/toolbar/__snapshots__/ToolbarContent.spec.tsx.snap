// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ToolbarContent /> SHOULD render 1`] = `
<div>
  <div
    class="MuiTypography-root MuiTypography-h6 MuiTypography-noWrap css-1futezt-MuiTypography-root"
  >
    Element Home Task
  </div>
</div>
`;
